### ISSUE TITLE
only 6 soldiers allowed for advent region base assault

### ISSUE THREAD

**Issue by bobjjones** [long-war-2/lwotc#1004]
I have reached the advent base assault to liberate a region. However, when I try to assault the facility, I am only able to send 6 soldiers. On either side of the 6 soldiers, there is an icon that says increase squad size in the GTS. As I understand it, those are not supposed to be there, and the mission is supposed to allow 10 soldiers. Given the number of enemies, I don't think I can do it with just 6. I normally have the ability to cycle around and send up to 10 people (though the infiltration timers make it so I can't). However, for this mission, squad select looks like vanilla. I have a save file, but I have no way to post it. Here is a screenshot. To show that I have the squad select mod properly active, I am attaching another screen shot showing my selected options for that mod. I found that disable infinite scrolling helped decrease the frequency of crashes.

![20200512233614_1](https://user-images.githubusercontent.com/38798278/81768663-90538d00-94a9-11ea-9cc0-68bde3000cc1.jpg)

![20200512232423_1](https://user-images.githubusercontent.com/38798278/81768474-220eca80-94a9-11ea-9fa0-13febdb29c3b.jpg)

**Comment by bobjjones** [comment]
ok, there are conflicting sources here.
On the geoscape, it says max squad size is 10. The UFOpedia says the max squad size is 10. But, when I watch you tube videos of people doing advent base assaults, some have 6 and some have 10. I can't figure out why the difference.

**Comment by bobjjones** [comment]
Seems this mod is dead. I'm just gonna play LW2. The chosen don't add much anyway and the lost are annoying.

**Comment by Grobobobo** [comment]
Your installation seems to be incorrect, no squad size upgrade should be available.

**Comment by Grobobobo** [comment]
Looking at the screen, you seem to not have robojumper's squad select installed despite it being in the required mod list.

**Comment by bobjjones** [comment]
I do have it installed. I showed a screen shot of the options for that mod. I wouldn't be able to select the options for a mod I don't have installed.

**Comment by bobjjones** [comment]
I was able to fix the problem myself by deleting all my config files and letting them get regenerated on the next start like I did when I first installed the mod.

But that means that something messed up my config files since my original installation or made it so they squad select didn't load properly.

I can perhaps chalk it up to a side-effect of all the crashes I get when I play the game. At least I have a work around for it now.
